firestore/firestore_snippets/partition_query_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+
+	"cloud.google.com/go/firestore"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestPartion(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	ctx := context.Background()
+	projectID := os.Getenv("GOLANG_SAMPLES_FIRESTORE_PROJECT")
```

Comment:
I think, we could use just `tc.ProjectID` here

Reply:
I guess we should keep it as it is, see [this comment](https://github.com/GoogleCloudPlatform/golang-samples/pull/1618#discussion_r480078594)